src/utils/transactions.ts

```diff
@@ -86,10 +86,10 @@ export function sortTransactions(a: HasTimestamp, b: HasTimestamp): number {
 }
 
 export const hasOnlyInputsWith = (inputs: Input[], addresses: Address[]): boolean =>
-  inputs.every((i) => addresses.map((a) => a.hash).indexOf(i.address) >= 0)
+  inputs.every((i) => addresses.map((a) => a.hash).indexOf(i?.address ?? '') >= 0)
```

Comment:
This looks cleaner to me

```suggestion
  inputs.every((i) => i?.address && addresses.map((a) => a.hash).indexOf(i.address) >= 0)
```

Reply:
Same for below